pipes/config.yaml

```diff
@@ -55,7 +55,7 @@ diamond_db: "/idi/sabeti-scratch/yesimon/db/diamond_db/nr"
 krona_db: "/idi/sabeti-scratch/yesimon/db/krona_taxonomy"
 
 # BWA index prefix for metagenomic alignment.
-align_rna_db: "/idi/sabeti-scratch/yesimon/db/rna_bwa/rna_bwa"
+align_rna_db: "/idi/sabeti-scratch/yesimon/db/rna_bwa/human_viral_16s"
```

Comment:
Should this, the krona db, and the taxonomy db point to databases stored in the newly-created `/idi/sabeti-scratch/shared-resources/`? (I see you already have an `rna_bwa` directory there.)